releaseNotes/AzureAppConfigurationProviderPython.md

```diff
@@ -2,6 +2,16 @@
 
 [Source code][source_code] | [Package (Pypi)][package] | [Samples][samples]
 
+## 2.0.0b2 - October 14, 2024
+
+### Features
+
+* Added AllocationId to Feature Flags when they have telemetry enabled.
```

Comment:
The way we put it made it sound like it's a new API. Suggest something like this:

Added support for including `AllocationId` to feature flag telemetry metadata when telemetry is enabled.